content/en/blog/2023-10-12-divvi-up-live-in-firefox.md

```diff
@@ -0,0 +1,25 @@
+---
+author: Bran Pitman
+date: 2023-10-11T00:00:00Z
+slug: divvi-up-live-in-firefox
```

Comment:
(pointed out by @jbr) Given that this announcement is for Firefox going live _soon_, this slug may be premature. We could drop the word `live`, perhaps?

Reply:
Changing to `divvi-up-in-firefox`